src/libraries/System.Net.Security/tests/FunctionalTests/SslStreamNetworkStreamTest.cs

```diff
@@ -3,11 +3,12 @@
 // See the LICENSE file in the project root for more information.
 
 using System.Net.Sockets;
+using System.Net.Test.Common;
 using System.Security.Authentication;
 using System.Security.Cryptography.X509Certificates;
+using System.IO;
```

Comment:
nit: `using System.IO` should appear in alphabetical order and thus be first in the list of using's.

Reply:
fixed